tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 360x360 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#let's do some posterior predictive checks to see if the\n",
```

Comment:
I think it would be illustrative to first show the actual posterior and say that it should center on `x_o` because everything is Gaussian here (maybe even plot the true posterior on top). 

afterwards you can show the predictive, and then motivate SBC because we additionally want to check the posterior variances.